packages/web-components/fast-components/src/styles/size.ts

```diff
@@ -5,3 +5,6 @@
  */
 export const heightNumber =
     "(var(--base-height-multiplier) + var(--density)) * var(--design-unit)";
+
+export const glyphSize =
```

Comment:
I know we talked about this calculation, but I didn't realize it was going to be shared. We'll need to remove density from the icon system, so let's not include this in here now. This looks like the calculation for the checkbox / radio size though, not the icon.

Reply:
moved into tree-item style file